pkg/gql/httpserver.go

```diff
@@ -16,6 +18,7 @@ import (
 	logger "github.com/sirupsen/logrus"
 
 	cfg "github.com/dusk-network/dusk-blockchain/pkg/config"
+	cryptoUtils "github.com/dusk-network/dusk-blockchain/pkg/util/crypto"
```

Comment:
Generally, it is more idiomatic to import packages with all lowercase. So it should be imported as `cryptoutils`

Reply:
Fixed